src/components/Navigation.tsx

```diff
@@ -21,7 +21,9 @@ const Navigation: FC = () => {
   const [projectName, setProjectName] = useState(
     project && !isLoading ? project.name : "default",
   );
-  const hasOidcCookie = Boolean(getCookie("oidc_access"));
+  const hasOidcCookie = Boolean(
```

Comment:
so I can login via OIDC now, but this never returns true for me which mean I can't logout via the UI. Not sure if I'm missing some oidc configs in lxd?

Reply:
Thank you for the QA. Upstream changed the cookie to be httponly, so we can't read it any more. I reworked the auth bits, so they become compatible to that change.